Part one answer

sky192 opened the pull request.

Kayla-SA-W commented:
Hey Samaheh,

This actually doesnt meet the requirements for part one: _the only requirements are that it provide documentation for using the API you will be building and that it be built using Pug_

I dont see any documentation, you can either add it to your README or to the page itself. Preferably both*

sky192 commented:
> Hey Samaheh,
> 
> This actually doesnt meet the requirements for part one: _the only requirements are that it provide documentation for using the API you will be building and that it be built using Pug_
> 
> I dont see any documentation, you can either add it to your README or to the page itself. Preferably both*

hi Kayla
I just changed my api actually I deleted my previous JSON file and made new one named coffee api
and added documentation.

Kayla-SA-W commented:
The first part of this assignment was to create a front end for your api with documentation that tell me about the routes of your assignment. When I run this I dont see your front end using pug and I dont see any documentation.

Grade: 0

sky192 commented:
> The first part of this assignment was to create a front end for your api with documentation that tell me about the routes of your assignment. When I run this I dont see your front end using pug and I dont see any documentation.
> 
> Grade: 0

It's weird but when i run http://localhost:1337/ it shows a front end using pug :(

Kayla-SA-W commented:
@sky192 okay i'll take a look at it again later when I grade part 2

sky192 commented:
> @sky192 okay i'll take a look at it again later when I grade part 2

thank you because I just run it and it worked

Kayla-SA-W commented:
Grade: 100
